pkg/oc/lib/env/envresolve.go

```diff
@@ -0,0 +1,168 @@
+package env
```

Comment:
See my other comment about this file and package. One of them needs to die, having two will cause confusion and may lead to code duplication.